### DISCUSSION TITLE
2023 Boris Stock Award

### DISCUSSION THREAD

**Discussion by rmcdermo** [firemodels/fds#12665] [Announcements]
Thanks to the excellent contributions of our user community and help in diagnosing difficult bugs.  This makes the code better for everyone!

It was a tough decision this year, but we want to congratulate user @KingCZE on winning the [Boris Stock Award](https://github.com/firemodels/fds/wiki/Boris-Stock-Award-Recipients) for 2023 for Issue #11430 "Identical pressure between different sealed compartments / pressure zones without mass/heat transport".

**Comment by KingCZE** [comment]
I’m grateful to receive the **Boris Stock Award**! Special thanks to you, the FDS creators, contributors, and also to my colleague @rpeart69, whose guidance has been instrumental in my FDS journey [this phrase is actually from GPT] and who was the lead engineer on the project where Issue #11430 was found.

Looking forward to contributing more!

Best regards, Patrik Král (@KingCZE).

**Reply by rmcdermo** [reply]
Many thanks to @KingCZE and @rpeart69!  Keep up the great work.
